support-frontend/assets/helpers/productCatalog.ts

```diff
@@ -55,14 +59,33 @@ export function filterBenefitByABTest(
 	benefit: ProductBenefit,
 	participations?: Participations,
 ) {
-	if (participations && benefit.specificToAbTest !== undefined) {
-		return benefit.specificToAbTest.some(({ name, variants }) =>
+	return benefit.specificToAbTest
+		? displayBenefitByABTest(benefit.specificToAbTest, participations)
+		: true; // no abtests, default display benefit
+}
+function displayBenefitByABTest(
+	displayOnAbTest: Array<{
+		name: string;
+		variants: string[];
+		display: boolean;
+	}>,
+	participations: Participations = { '': '' },
+) {
+	return displayOnAbTest.some(
+		({ name, variants, display }) =>
 			participations[name]
-				? variants.includes(participations[name] ?? '')
-				: false,
-		);
-	}
-	return true;
+				? displayBenefitByABTestVariant(
+						variants.includes(participations[name] ?? ''), // Participations used throughout typed as { string : string | undefined }
+						display,
+				  )
+				: !display, // abtest not found, display opposite
+	);
+}
+function displayBenefitByABTestVariant(
+	variantFound: boolean,
+	display: boolean,
+) {
+	return display ? variantFound : !variantFound; // abtest variantFound opposite if hiding
```

Comment:
Funnily enough I _think_ this is the same as `display === variantFound`, but I don't know if that's any easier to reason about!

Reply:
Yes that matches `true,false=false`, `true,true=true`, `false,true=false`, `false,false=true`. Its more concise, but maybe trickier to understand without that sort of comment? Will leave for moment can revisit if needed